config/base.go

```diff
@@ -26,8 +28,23 @@ func save(filePath string, cfg interface{}) error {
 	return encoder.Encode(cfg)
 }
 
-// load reads file, parses json and stores in cfg struct
-func load(filePath string, cfg interface{}) error {
+// Load reads file, parses json and stores in cfg struct
+func Load(filePath string, cfg interface{}) error {
```

Comment:
It is too public to be exposed. Consider putting this function into an `internal` package.

Reply:
There is no good way to do this without refactoring how confg.json and signingkeys.json are read/modified and thats the reason I am going to leave this method private and duplicate code in trustpolicy.go file